packages/react-components/src/components/Switch/Switch.module.scss

```diff
@@ -188,4 +188,11 @@ $medium-move: $medium-width - $medium-size;
       border-width: 2px;
     }
   }
+
+  &__disabled-icon {
```

Comment:
Shouldn't we add `xxsmall` size for the icon?
https://github.com/livechat/design-system/blob/294d06d5eab1570d29db59e8d0f1e47c2892d33e/packages/react-components/src/components/Icon/Icon.tsx#L16-L37

Reply:
It turned out the icon was wrong -> after changing the lock icon [094a699](https://github.com/livechat/design-system/pull/372/commits/094a6996224c3cc7d832d5e27cc66826f4da7fc5), everything is working as expected and this specific CSS size is removed